var/spack/repos/builtin/packages/avro/package.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import inspect
+import os
+
+from llnl.util import filesystem as fs
+
+from spack.package import *
+
+
+class Avro(CMakePackage):
+    """Apache Avro data serialization system."""
+
+    homepage = "https://www.apache.org/dyn/closer.cgi/avro/"
+    url = "https://dlcdn.apache.org/avro/avro-1.11.3/avro-src-1.11.3.tar.gz"
+
+    maintainers("teaguesterling")
+
+    license("APACHE-2.0", checked_by="teaguesterling")
+
+    version("1.11.3", sha256="6ea787a83260a11b5a899aadd22f701e24138477cd7bf789614051a449dcc034")
+
+    # Failing in examples for me
+    variant("c", default=False, description="Built the C library")
+    variant("cxx", default=True, description="Built the C++ library")
+    # TODO: java, javascript, perl, python, ruby, rust?
```

Comment:
Another package that has multiple language bindings: librsvg. (Surely there are more but that's the one I touched most recently.)

Reply:
I'll look. Thanks for the tip!